harvard-imperial-college-london.csl

```diff
@@ -25,7 +28,7 @@
   </macro>
   <macro name="author">
     <names variable="author">
-      <name and="symbol" name-as-sort-order="all" sort-separator=", " initialize-with="." delimiter-precedes-last="never" delimiter=", "/>
+      <name and="symbol" delimiter-precedes-last="never" et-al-min="19" initialize-with="." name-as-sort-order="all"/>
```

Comment:
we prefer the `et-al-min` to be on the bibliography instead of here, where it's hard to track down when editing/troubleshooting.